components/Input/Input.tsx

```diff
@@ -14,6 +14,8 @@ const Input: React.FC<InputProps> = ({
   keyboardType = 'default',
   secureTextEntry = false,
   onInputBlur,
+  multiline,
+  numberOfLines,
 }) => {
```

Comment:
There is a way to extend the TextInput props and provide them all together instead of adding new props when you need them, it's a combination of rest parameter and typescript's ability to extend types.